docs/source/gettingstarted.rst

```diff
@@ -188,6 +189,21 @@ They are:
 Any combination of these arguments can be given. Omitting them all is fine too.
 Their behavior is documented in detail below.
 
+``max_y``
+^^^^^^^^^
+
+Specifies the maximum active twitch force to be set in the output xlsx file.
+When specified, max_y sets all graphs in continuous-waveform-snapshot and full-continuous-waveform-plots to have a y-axis scale of 0 - max_y.
```

Comment:
We should word this the same as the descriptions for the other args:

`Specifies the max y-axis bound for all graphs in continuous-waveform-snapshot and full-continuous-waveform-plots. The min bound is always 0. By default, this parameter is set to the max tissue waveform value across all wells.`